server/protocols/webadmin/webadmin-mailrepository/src/main/java/org/apache/james/webadmin/routes/MailRepositoriesRoutes.java

```diff
@@ -59,6 +72,50 @@ public void define(Service service) {
         this.service = service;
 
         defineGetMailRepositories();
+
+        defineListMails();
+    }
+
+    @GET
+    @Path("/{encodedUrl}/mails")
+    @ApiOperation(value = "Listing all mails in a repository")
+    @ApiImplicitParams({
+        @ApiImplicitParam(
+            required = false,
+            paramType = "query parameter",
+            dataType = "Integer",
+            defaultValue = "0",
+            example = "?offset=100",
+            value = "If present, skips the given number of key in the output."),
+        @ApiImplicitParam(
+            required = false,
+            paramType = "query parameter",
+            dataType = "Integer",
+            defaultValue = "0",
```

Comment:
I doubt it's 0 (which is invalid)

Reply:
That must be a rebase fail. Latest value is "absent"